src/views/Home.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/mesh-logo.png">
+    <router-link to="/about">
+      <img class="rotating" alt="Vue logo" src="../assets/mesh-logo.png">
```

Comment:
Nice. We'll want to update the alt text I think.

Reply:
You're totally right.  Will do.